js/__loader.js

```diff
@@ -283,6 +288,22 @@
 
   global.process = loader.require('process');
   global.Buffer = loader.require('buffer').Buffer;
+  const stream = loader.require('stream');
+  class StdoutStream extends stream.Writable {
+    _write(chunk, encoding, callback) {
+      runtime.stdio.defaultStdio.write(String(chunk));
+      callback();
+    }
+  }
+  class StderrStream extends stream.Writable {
+    _write(chunk, encoding, callback) {
+      runtime.stdio.defaultStdio.writeError(String(chunk));
+      callback();
+    }
+  }
```

Comment:
Actually I mean that this changes are in es6 style while the whole file is in es5. Just a note. That's ok, I suppose.